content/install_maintain/installation/installing-cluster.md

```diff
@@ -221,24 +179,93 @@ postgresql_server:
         # Should be the same on all nodes
         postgres:
             replicator_password: '<strong password for replication user>'
+  
 
-    enable_remote_connections: true
-    ssl_enabled: true
-    ssl_only_connections: true
+# For monitoring service(status reporter)
+prometheus:
+  credentials:
+    username: <monitoring username>
+    password: <srtong password for monitoring user>
 
-    # If true, client certificate verification will be required for PostgreSQL clients,
-    # e.g. Cloudify Management service cluster nodes.
-    ssl_client_verification: false
+  cert_path: <certificate for prometheus, cert_path of postgresql_server can be used>
+  key_path: <key for promethus, key_path of postgresql_server can be used>
+  ca_path: <ca for promethus, key_path of postgresql_server can be used>
+  postgres_exporter:
+    # `password` is a placeholder and will be updated during config file rendering, based on postgresql_server.postgres_password
+    password: ''
+    sslmode: require
 
 services_to_install:
     - database_service
+    - monitoring_service
 ```
 
-Execute on each node by order:
+Execute on each node **sequentially** (i.e. do not start installing next manager unless the previous has been successfully installed):
 ```bash
 cfy_manager install [--private-ip <PRIVATE_IP>] [--public-ip <PUBLIC_IP>] [-v]
+``` 
+
+On one node, verify that everything looks healthy with: `cfy_manager dbs list` .
+
+Example:
+```yaml
+manager:
+  private_ip: <ip of this host>
+  public_ip: <ip of this host>
+
+postgresql_server:
+  postgres_password: 'areallystrongandsecretpasswordforpostgres'
+  
+  cert_path: '/home/centos/.cloudify-test-ca/<ip of this host>.crt'
+  key_path: '/home/centos/.cloudify-test-ca/<ip of this host>.key'
+  ca_path: '/home/centos/.cloudify-test-ca/ca.crt'
+  
+  cluster:
+    nodes:
+      <first postgresql instance-name>:
+        ip: <private ip of postgres server 1>
+      <second postgresql instance-name>:
+        ip: <private ip of postgres server 2>
+      <third postgresql instance-name>:
+        ip: <private ip of postgres server 3>
+
+    etcd:
+      cluster_token: 'astrongandsecretpasswordlikestring'
+      root_password: 'anotherstrongandsecretbutdifferentpassword'
+      patroni_password: 'yetanotherstrongandsecretpassword'
+
+    patroni:
+      rest_user: patroni
+      rest_password: 'strongandsecretpatronirestpassword'
+
+    postgres:
+      replicator_password: 'stillanotherstrongandsecretpassword'
+
+# For monitoring service(status reporter)
+prometheus:
+  credentials:
+    username: 'monitoringusername'
+    password: 'longyeteasytorememberstringasapassword'
+
+  cert_path: '/home/centos/.cloudify-test-ca/<ip of this host>.crt'
+  key_path: '/home/centos/.cloudify-test-ca/<ip of this host>.key'
+  ca_path: '/home/centos/.cloudify-test-ca/ca.crt'
+  postgres_exporter:
```

Comment:
I'm hoping we can clean up this section of the example once the comments in the other copy of this section are dealt with. Nothing to do with it until the comments above are handled anyway.